### PR TITLE
Fix Regression Linux i386 workflow

### PR DESCRIPTION
**Pull request by mfundul**
Change the postgresql ftp URL from snapshot/ to source/.

### CODE REVIEW

File: .github/workflows/linux-32bit-build-and-test.yaml [hunk 1]
```diff
@@ -19,13 +19,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pg: [ "12", "13" ]
+        pg: [ "12.7", "13.3" ]
         build_type: [ Debug ]
         include:
-          - pg: 12
+          - pg: "12.7"
```
Comment (mkindahl) [inline .github/workflows/linux-32bit-build-and-test.yaml:25]:
```suggestion
          - pg: 12.7
```

File: .github/workflows/linux-32bit-build-and-test.yaml [hunk 2]
```diff
@@ -19,13 +19,15 @@ jobs:
...
             ignores: append-12 chunk_adaptive continuous_aggs_bgw_drop_chunks remote_txn transparent_decompression-12 plan_skip_scan-12
-          - pg: 13
+            pg_major: "12"
+          - pg: "13.3"
```
Comment (mkindahl) [inline .github/workflows/linux-32bit-build-and-test.yaml:28]:
```suggestion
            pg_major: 12
          - pg: 13.3
```

File: .github/workflows/linux-32bit-build-and-test.yaml [hunk 3]
```diff
@@ -19,13 +19,15 @@ jobs:
...
             ignores: append-13 chunk_adaptive remote_txn transparent_decompression-13 vacuum_parallel plan_skip_scan-13
+            pg_major: "13"
```
Comment (mkindahl) [inline .github/workflows/linux-32bit-build-and-test.yaml:30]:
```suggestion
            pg_major: 13
```

**Review by mkindahl — COMMENTED** [review]
Just nits, but please explain in the commit message *why* it is necessary to change the FTP URL not just *what* the commit does.

**Review by mkindahl — APPROVED** [review]

**Review by svenklemm — APPROVED** [review]

**Pull request merged**